docs/src/modules/spring/pages/workflow-entities.adoc

```diff
@@ -1,6 +1,6 @@
 = Implementing Workflow Entities
 :page-aliases: spring:workflow.adoc
-:sample-url: TODO
+:sample-url: https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples/spring-transfer-workflow
```

Comment:
Is this used somewhere?

Reply:
OK, in that case, I think you can use this var in line `#110` .. see the example on that .adoc

Reply:
Nice, I have no idea why IntelliJ did show me this var usage, anyway, thanks.